doc/install.asciidoc

```diff
@@ -51,11 +51,23 @@ Download the https://packages.debian.org/sid/all/qutebrowser/download[qutebrowse
 https://packages.debian.org/sid/all/python3-pypeg2/download[PyPEG2]
 package from the Debian repositories.
```

Comment:
If we remove the wget commands, this should probably explain what to download (-qtwebkit/-qtwebengine).